spacy/cli/init_config.py

```diff
@@ -83,6 +83,7 @@ def init_fill_config_cli(
     # fmt: off
     base_path: Path = Arg(..., help="Path to base config to fill", exists=True, dir_okay=False),
     output_file: Path = Arg("-", help="Path to output .cfg file (or - for stdout)", allow_dash=True),
+    distillation: bool = Opt(False, "--distillation", "-pt", help="Include config for distillation (with 'spacy distill')"),
```

Comment:
Needs a different shortcut, `-pt` is c/p from `pretraining`

Reply:
Not sure how I missed that. Changed to `-d`.

Reply:
You don't think `-d` and `-D` are too similar? For only a small set of options it seems unfortunate to reuse the same single letter maybe?

Reply:
maybe `-dt`? 🤷

Reply:
My unix brain can't parse multi-letter single dash option 😄 😓 , but changed...

Reply:
Yea I get that! Sorry 🙈